acc.config.json

```diff
@@ -68,11 +69,14 @@
   "fetchQuestionsFromBackend": false,
   "theme": {},
   "showVulaanControls": true,
+  "useAreasInsteadOfStadsdeel": false,
+  "areaTypeCodeForDistrict": "stadsdeel",
```

Comment:
In the schema, this field is marked as required which, in my opinion, shouldn't be the case. Setting the prop `useAreasInsteadOfStadsdeel` to 'false' makes `areaTypeCodeForDistrict` useless and thus shouldn't have a value. Let's wait with merging this request till the PR in Amsterdam/signals-frontend has been updated.